### DISCUSSION TITLE
not sure if this is a bug for cozy or for vlads

### DISCUSSION THREAD

**Discussion by zethfoxster** [vladmandic/sdnext#1825] [Q&A] [answered]
![image](https://github.com/vladmandic/automatic/assets/5951657/3c64ca46-cc01-49e4-8268-4ebf4e1883c2)

when using cozy, in inpaint, this little info box makes it impossible to try and inpaint without having to stretch the canvas far away from it....can this info button/box possibly be moved over by the tools instead of on the left side? better still, can it be made click to show instead of hover to show.

my mouse cursor in that screenshot is located on her eye.... no where near the i icon....

**Comment by vladmandic** [comment, marked as answer]
those strings do not appear in sdnext codebase, so it must be external.

**Reply by zethfoxster** [reply]
found it and removed the offending extension. it was an extension called canvas zoom...
